andela/suyabay
# Remove subscription count from channels table

#### What does this PR do?

Remove subscription count from channels table
#### Description of Task to be completed?
- Remove subscription_count from create channels migration file  and,
- Create a new migration file to remove the subscription_count from channels table
#### How should this be manually tested?
- From the admin dashboard, click on create channel, you should see channel created successfully and not an error message that "subscription_count " field does not exist
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#119172335
#### Screenshots (if appropriate)
#### Questions:

[@andela-oadebayo](https://github.com/andela-oadebayo)
[@unicodeveloper](https://github.com/unicodeveloper)
